### ISSUE TITLE
[$100] Investigate new error

### ISSUE THREAD

**Issue by jmgasper** [topcoder-platform/evernym-tc-wallet#15]
@seriyvolk83:

With the latest code in Github, we're seeing this error when running the app.  Any thoughts on why that might be?  It wasn't doing that for me previously, and I don't see any changes that could affect this.

```
2020-12-18 19:46:11.427775+1100 EvernymApp[17587:891187] Fatal error: The operation couldn’t be completed. (VcxErrorDomain error 1025.): file EvernymApp/AppDelegate.swift, line 213
```

<img width="616" alt="Screen Shot 2020-12-18 at 7 47 24 pm" src="https://user-images.githubusercontent.com/21790/102594366-3404ca00-416a-11eb-8cab-2693a34f29df.png">

```
Agency config         {
        "agency_url": "https://agency.pstg.evernym.com",
        "agency_did": "LqnB96M6wBALqRZsrTTwda",
        "agency_verkey": "BpDPZHLbJFu67sWujecoreojiWZbi2dgf4xnYemUzFvB",
        "wallet_name": "Topcoder-Dev",
        "wallet_key": "JIKh7cwxsqEioxPOJmvSjSItWyE4bzIwzqVBT7+wGB0AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="
        }
sdkApi.agentProvisionAsync...
Success: agentProvisionAsync: oneTimeInfo: Optional("{\"agency_did\":\"LqnB96M6wBALqRZsrTTwda\",\"agency_endpoint\":\"https://agency.pstg.evernym.com\",\"agency_verkey\":\"BpDPZHLbJFu67sWujecoreojiWZbi2dgf4xnYemUzFvB\",\"genesis_path\":\"<CHANGE_ME>\",\"institution_did\":\"Pdwkx7omPSFciaKjL5AfPW\",\"institution_logo_url\":\"<CHANGE_ME>\",\"institution_name\":\"<CHANGE_ME>\",\"institution_verkey\":\"DLbkaNy2a6wHGiSg2wT7S3mX3paBDrk9LpA7TmrWbjhb\",\"protocol_type\":\"1.0\",\"remote_to_sdk_did\":\"VHaKRkULKCujEFgGTUuwaD\",\"remote_to_sdk_verkey\":\"GRAnxUnFPhvVp7LuRevKp8FhPBKJ4jw9BvcpEN8j8BNZ\",\"sdk_to_remote_did\":\"Pdwkx7omPSFciaKjL5AfPW\",\"sdk_to_remote_verkey\":\"DLbkaNy2a6wHGiSg2wT7S3mX3paBDrk9LpA7TmrWbjhb\",\"wallet_key\":\"JIKh7cwxsqEioxPOJmvSjSItWyE4bzIwzqVBT7+wGB0AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=\",\"wallet_name\":\"Topcoder-Dev\"}")
Creating pool transaction genesis file was successful: /var/mobile/Containers/Data/Application/3D852C8E-509E-465A-BF67-02CE49607ABD/Documents/pool_transactions_genesis_STAG_2
Updated config: {
  "pool_name" : "7e96cbb3b0a1711f3b843af3cb28e31dcmpool",
  "wallet_key" : "JIKh7cwxsqEioxPOJmvSjSItWyE4bzIwzqVBT7+wGB0AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=",
  "genesis_path" : "\/var\/mobile\/Containers\/Data\/Application\/3D852C8E-509E-465A-BF67-02CE49607ABD\/Documents\/pool_transactions_genesis_STAG_2",
  "protocol_version" : "2",
  "institution_did" : "Pdwkx7omPSFciaKjL5AfPW",
  "remote_to_sdk_did" : "VHaKRkULKCujEFgGTUuwaD",
  "agency_verkey" : "BpDPZHLbJFu67sWujecoreojiWZbi2dgf4xnYemUzFvB",
  "protocol_type" : "1.0",
  "agency_did" : "LqnB96M6wBALqRZsrTTwda",
  "agency_endpoint" : "https:\/\/agency.pstg.evernym.com",
  "wallet_name" : "Topcoder-Dev",
  "institution_name" : "real institution name",
  "remote_to_sdk_verkey" : "GRAnxUnFPhvVp7LuRevKp8FhPBKJ4jw9BvcpEN8j8BNZ",
  "institution_verkey" : "DLbkaNy2a6wHGiSg2wT7S3mX3paBDrk9LpA7TmrWbjhb",
  "sdk_to_remote_verkey" : "DLbkaNy2a6wHGiSg2wT7S3mX3paBDrk9LpA7TmrWbjhb",
  "sdk_to_remote_did" : "Pdwkx7omPSFciaKjL5AfPW",
  "institution_logo_url" : "https:\/\/robothash.com\/logo.png"
})
sdkApi.initWithConfig...

ERROR [initWithConfig]: Optional(Error Domain=VcxErrorDomain Code=1025 "(null)")
Fatal error: The operation couldn’t be completed. (VcxErrorDomain error 1025.): file EvernymApp/AppDelegate.swift, line 213

2020-12-18 19:46:11.427775+1100 EvernymApp[17587:891187] Fatal error: The operation couldn’t be completed. (VcxErrorDomain error 1025.): file EvernymApp/AppDelegate.swift, line 213
(lldb)

**Comment by jmgasper** [comment]
Contest https://www.topcoder.com/challenges/30159064 has been created for this ticket.<br/><br/>```This is an automated message for ghostar via Topcoder X```

**Comment by jmgasper** [comment]
Contest https://www.topcoder.com/challenges/30159064 has been updated - it has been assigned to seriyvolk83.<br/><br/>```This is an automated message for ghostar via Topcoder X```

**Comment by seriyvolk83** [comment]
That's very strange. The only difference is Bundle ID. I tried different bundle IDs and some of them work fine, and some fails with 1025 error.

Work fine:
com.topcoder.EvernymApp
com.topcoder.Evernym
com.topcoder.Ever

Not work:
com.topcoder.ios.evernym-wallet
com.topcoder.EvernymB
com.topcoder.EvernymC
com.topcoder.EvernymA
com.topcoder.EvernymAp

Seems any prefix for `com.topcoder.EvernymApp` works fine.

I changed `walletName` for all cases, and it does not affect the result.

I can't understand why Bundle ID change affects the result.

I can only guess that when connected successfully, something is cached in the service and later attempts work.

I suggest to change the bundle ID to shorter name and change `walletName` (!) before running the app with new bundle id.
